microSALT/utils/job_creator.py

```diff
@@ -13,46 +13,56 @@
 import time
 
 from datetime import datetime
-from microSALT.store.lims_fetcher import LIMS_Fetcher
 from microSALT.store.db_manipulator import DB_Manipulator
+from microSALT.utils.referencer import Referencer
 
 class Job_Creator():
 
-  def __init__(self, input, config, log, finishdir="", timestamp="", trim=True, qc_only=False,careful=False,pool=list()):
+  def __init__(self, config, log, sampleinfo={}, run_settings={}):
     self.config = config
     self.logger = log
     self.batchfile = "/tmp/batchfile.sbatch"
```

Comment:
Is it a good idea to hardcode the path to /tmp like this? Maybe you could use the `tempfile` module in python to create temporary files?